### ISSUE TITLE
Add diff_flashlight

### ISSUE THREAD

**Issue by uzervlad** [ppy/osu-api#322]
As there is a [new strain](https://github.com/ppy/osu/blob/master/osu.Game.Rulesets.Osu/Difficulty/Skills/Flashlight.cs) skill, it should be added to the API

**Comment by peppy** [comment]
Is this not exposed by the v2 API already? If so, I don't think we will be exposing new attributes here.

**Comment by uzervlad** [comment]
No, v2 doesn't return any information about separate skills.

**Comment by peppy** [comment]
That's a bit of an oversight. We'll need to add that eventually, but I'll see that this is added to the v1 API in the mean time.

**Comment by peppy** [comment]
Before I add this, what is the use case for using this?

**Comment by uzervlad** [comment]
Just PP calculation, I don't think it can be used for anything else.

**Comment by uzervlad** [comment]
Just PP calculation. I don't think it can be used for anything else.
It's too hard to write and debug star rating calculation on my own, so I use values from API

**Comment by peppy** [comment]
Hmm, we already have tools which you can run to produce these locally, have you tried using those? I think that would be more efficient in most cases.

I'm just asking because I think 99% of use cases won't use this attribute and adding it bloats the response and the lookup overhead, potentially for no reason.

**Comment by TheEZIC** [comment]
I also need this, because I use it in my front-end only extension to calculate PP 
[github repository](https://github.com/TheEZIC/O-pp)
[google extension store page](https://chrome.google.com/webstore/detail/opp/hjlbhnjnmmfadkobjkmddmgopanllkna?hl=ru&)
I can't use v2 API. I don't remember exactly why, but the issue was related to converted beatmaps.

**Comment by uzervlad** [comment]
> Hmm, we already have tools which you can run to produce these locally, have you tried using those?

Yes, I have, a rewrite of my bot uses lazer to calculate star rating and PP. But the current version is relying on values from API.
Actually, while writing this I realized that it shouldn't be too hard to connect lazer with the existing codebase.

@TheEZIC's use case, on the other hand, probably can't rely on any external tools, so if your only concerns are bloat and lookup overhead, you could add a query parameter for showing all difficulty attributes.
